src/GeekPlatform/ViewModels/Profil/ProfilViewModel.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using GeekPlatform.Models;
+
+namespace GeekPlatform.ViewModels.Profil
+{
+    public class ProfilViewModel : ViewModelBase
+    {
+        public ProfilViewModel(Profile user) : base(user)
+        {
+            Nev = user.Name;
+            PozicioTeamTagsag = user.TeamMember;
```

Comment:
Így is tudjuk magyarosan formázni amúgy, view-ben nyugodtan lehet paraméteresen tostringelni. (Persze nem mondom hogy az jobb megoldás, egyelőre az egyszerűség miatt én is efelé hajlok csak meg akartam jegyezni hogy ebben is van ráció.)